ACCS-533: Documentation for Card Vaulting - Save Card Checkbox (#804)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/payment-services/containers/credit-card.mdx

</TableWrapper>

## Card vaulting

> The CreditCard container displays a "Save this card for future purchases" checkbox when both conditions are met:

Reviewer: ```suggestion
The CreditCard container shows a Save this card for future purchases checkbox when both of the following are true:
```

> - The shopper is **logged in** (determined by the `getCustomerToken` function provided during [initialization](/dropins/payment-services/initialization/))

Reviewer: ```suggestion
- The shopper is **logged in** as indicated by the `getCustomerToken` function you provide at [initialization](/dropins/payment-services/initialization/)
```

> - Card vaulting is **enabled** in the Adobe Commerce Admin under **Stores > Configuration > Sales > Payment Methods > Adobe Payment Services > Credit Card Fields > Vault Enabled**

Reviewer: ```suggestion
- Card vaulting is **enabled** in the Adobe Commerce Admin at **Stores > Configuration > Sales > Payment Methods > Adobe Payment Services > Credit Card Fields > Vault Enabled**
```

> When the shopper checks the box and submits, the card is stored for future purchases. The checkbox is automatically hidden for guest shoppers or when vaulting is disabled.

Reviewer: ```suggestion
If the shopper selects the checkbox and submits, the card is saved for future purchases. The checkbox is hidden automatically for guest shoppers and when vaulting is disabled.
```

> If the shopper selects the checkbox and submits, the card is saved for future purchases. The checkbox is hidden automatically for guest shoppers and when vaulting is disabled.

Reviewer 2: Maybe "The checkbox is automatically hidden when either the shopper is a guest or vaulting is disabled" is clearer than
"The checkbox is hidden automatically for guest shoppers and when vaulting is disabled"?

Just to be sure that the checkbox will be hidden when any of the two conditions are met, not both at the same time :)

Author: Applied, thanks!
